spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/TaskTemplate.java

```diff
@@ -76,6 +76,8 @@ public class TaskTemplate implements TaskOperations {
 
 	private static final String RETRIEVE_LOG = "tasks/logs";
 
+	private static final String RETRIEVE_LOG_TASK_EXECUTION_ID = "tasks/logs/taskexecutionid";
```

Comment:
I would be in favor of renaming the existing constant, creating the pair `RETRIEVE_LOG_BY_EXTERNAL_ID` and `RETRIEVE_LOG_BY_INTERNAL_ID`.